check_sites.py

```diff
@@ -111,10 +115,14 @@ def main():
                 return
         old_checker = RwsCheck(old_sites, etlds, icanns)
         check_sets, subtracted_sets = find_diff_sets(old_checker.load_sets(), rws_checker.load_sets())
-        # TODO: add variable and check for subtracted_sets in case of user 
-        # removing old set from the list
     else:
         check_sets = rws_checker.load_sets()
+        if cli_primaries:
+            absent_primaries = [p for p in cli_primaries if p not in check_sets]
+            for p in absent_primaries:
+                error_texts.append("There was an error loading the set:\n" + 
+                    p + " could not be found in related_website_sets.JSON")  
+            check_sets = {p: check_sets[p] for p in cli_primaries if p not in absent_primaries}
```

Comment:
If one of the cli_primaries isn't in `absent_primaries`, that means it *is* in `check_sets`, so this feels a bit more direct/clear to me:

```suggestion
            check_sets = {p: check_sets[p] for p in cli_primaries if p in check_sets}
```

Reply:
Done.